tools/nfpm.yaml

```diff
@@ -0,0 +1,17 @@
+name: "logcli"
+arch: "amd64"
+platform: "linux"
+version: ${CIRCLE_TAG}
```

Comment:
What is `CIRCLE_TAG` and where is it defined?

Reply:
It's provided by the Circle CI job (https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables). It's already used by `tools/release-note.md` and `tools/release`.